src/Autofac/Core/Container.cs

```diff
@@ -100,6 +99,12 @@ public ILifetimeScope BeginLifetimeScope(object tag, Action<ContainerBuilder> co
             return _rootLifetimeScope.BeginLifetimeScope(tag, configurationAction);
         }
 
+        /// <inheritdoc/>
```

Comment:
My thinking was that since tracing can be attached per-scope, if you wanted to only do tracing temporarily, you could create a new lifetime scope, and trace just that scope.

Removing a tracer from a parent scope would not remove it from any child scopes (because we do not track parent -> child; so it would potentially be more confusing.

Reply:
I'll have to think more about the use case of tracing a specific scope. It's interesting, but I'm not sure how often it'd be used outside of, say, unit testing or something. Like, if I was diagnosing an issue in an ASP.NET Core app, the best I could do would be attach to the request lifetime scope, which is still every request.

I think it's fine for now. I'd like to investigate more about DiagnosticSource and the options it might open up. But that can be done later.